themes/default/content/resources/platform-with-aws-proton-and-pulumi/index.md

```diff
@@ -0,0 +1,62 @@
+---
+preview_image: ""
+hero:
+  image: /icons/containers.svg
+  title: Platform Engineering with AWS Proton and Pulumi
+main:
+  duration: 60 minutes
+  presenters:
+    - name: Josh Kodroff
+      role: Sr. Solutions Architect, Pulumi
```

Comment:
```suggestion
      role: Senior Solutions Architect, Pulumi
```
(They should be consistent between me and Adam, whichever representation we use.)